Main.playground/Contents.swift

```diff
@@ -6,28 +6,129 @@
 //
 import Foundation
 
-class Node{
+class Node: Hashable, Equatable , Comparable{
```

Comment:
When we are confirming to a protocol like the above we keep it seperate in an extension 
this way if there is extra functions needed for each one it can be separated from the main code 

extension Node: Hashable {}
extension Node: Equatable {}
extension Node: Comparable {}